docs/rfcs/text/YYYYmmDD-template.md

```diff
@@ -0,0 +1,22 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```

Comment:
Duplicate with title

Reply:
I removed "Start Date". "Feature Name" is also included in file name but I think we should keep it?

Reply:
Yeah, title in body is more accurate, filename is short and concise.